api/app/Models/User.php

```diff
@@ -156,67 +109,32 @@ class User extends Model implements Authenticatable, HasLocalePreference, Laratr
     protected $fillable = [
         'email',
         'sub',
-        'searchable',
     ];
 
-    protected $hidden = [
-        'searchable',
-    ];
+    protected $hidden = [];
 
-    public static function getSelectableColumns()
+    public function searchableOptions()
     {
-        return self::$selectableColumns;
+        return [
+            // You may want to store the index outside of the Model table
+            // In that case let the engine know by setting this parameter to true.
+            'external' => true,
+            // If you don't want scout to maintain the index for you
+            // You can turn it off either for a Model or globally
+            'maintain_index' => false,
```

Comment:
Updating that runs automatically was pointing at `user.searchable` and failing
So I added `static::updated()` below